Fix #6912 /mozilla/firefox/releases/3/templates/index.html (#6921)
mdn/translated-content: files/es/mozilla/firefox/releases/3/templates/index.md

---
title: Plantillas en Firefox 3
slug: Mozilla/Firefox/Releases/3/Templates
translation_of: Mozilla/Firefox/Releases/3/Templates
original_slug: Plantillas_en_Firefox_3
---
{{FirefoxSidebar}}

> Las plantillas se han mejorado enormemente en Firefox 3. La mejora más importante permite el uso de [procesador de consultas personalizado](/es/Cómo_implementar_un_componente_procesador_de_consultas_XUL) para manejar otro tipo de fuentes de datos además de RDF. Esto es posible gracias a la nueva sintaxis de consultas. También se incluyen soporte para SQL ([mozStorage](/es/docs/Storage)) y fuentes de datos XML. Ya {{ interwiki('wikimo', 'XUL:Template_Features_in_1.9', 'está disponible') }} una descripción completa de las nuevas características para plantillas. ({{ Bug(285631) }})

Reviewer: @JuanVqz Corrígeme si me equivoco, pero creo que las url quedan practicamente igual, solo se cambia el código locale ¿No?
```suggestion
Las plantillas se han mejorado enormemente en Firefox 3. La mejora más importante permite el uso de [procesador de consultas personalizado](/es/How_to_implement_a_custom_XUL_query_processor_component) para manejar otro tipo de fuentes de datos además de RDF. Esto es posible gracias a la nueva sintaxis de consultas. También se incluyen soporte para SQL ([mozStorage](/es/docs/Storage)) y fuentes de datos XML. Ya {{ interwiki('wikimo', 'XUL:Template_Features_in_1.9', 'está disponible') }} una descripción completa de las nuevas características para plantillas. ({{ Bug(285631) }})
```

### Otras mejoras

- Se han añadido condiciones relacionales para que los resultados encontrados cumplan la norma con una mayor exactitud. Esto permite, por ejemplo, que se encuentren resultados que empiecen o terminen con determinados hilos, o que son antes o después otros valores.
- Se ha incluido una bandera, `dont-recurse`, para evitar que se repitan resultados generados tan solo a un nivel.
- Se han añadido APIs al editor de plantillas para recuperar un objeto de resultado que representa un dato externo.

> - Se ha mejorado el servicio tipo XUL [sorts](/es/XUL/Guía_de_plantillas/Ordenar_resultados) clasificando mejor tanto el árbol de contenido como el de sin contenido. También permite clasificar contenidos de plantillas no editables. ({{ Bug(335122) }})

Reviewer: Mismo caso que la review anterior.
```suggestion
- Se ha mejorado el servicio tipo XUL [sorts](/es/XUL/Template_Guide/Sorting_Results) clasificando mejor tanto el árbol de contenido como el de sin contenido. También permite clasificar contenidos de plantillas no editables. ({{ Bug(335122) }})
```

### Véase también

> - [Firefox 3 para desarrolladores](/es/docs/Mozilla/Firefox/Releases/3)

Reviewer: Nuestra respectiva línea en blanco al final que por alguna razón Github se la ahorra jeje
```suggestion
- [Firefox 3 para desarrolladores](/es/docs/Mozilla/Firefox/Releases/3)

```

---
title: Plantillas en Firefox 3
slug: Mozilla/Firefox/Releases/3/Templates
translation_of: Mozilla/Firefox/Releases/3/Templates

> original_slug: Plantillas_en_Firefox_3

Reviewer 2: ```suggestion
original_slug: Plantillas_en_Firefox_3
l10n:
  sourceCommit: 2aaa075dd310f0de930bab617bb6fe1033970da3
```

Ese sería el sha que tiene que agregar

Author: Muchas gracias @marcelozarate, tremenda tu ayuda! :D

> ---

Reviewer 3: no fui capaz de encontrar el archivo en ingles que corresponde a este documento (en la web).

```suggestion
l10n:
  souceCommit: sha-del-archivo-en-ingles
---
```

---
title: Plantillas en Firefox 3
slug: Mozilla/Firefox/Releases/3/Templates
translation_of: Mozilla/Firefox/Releases/3/Templates
original_slug: Plantillas_en_Firefox_3

> l10n:
>   sourceCommit: 2aaa075dd310f0de930bab617bb6fe1033970da3
> l10n:
>   souceCommit: sha-del-archivo-en-ingles

Reviewer: Eso se coló por ahí jeje
```suggestion
l10n:
  sourceCommit: 2aaa075dd310f0de930bab617bb6fe1033970da3
```
